privacyidea/lib/realm.py

```diff
@@ -154,7 +154,7 @@ def delete_realm(realmname):
     defRealm = get_default_realm()
     hadDefRealmBefore = (defRealm != "")
 
-    realm = Realm.query.filter_by(name=realmname).first()
+    realm = fetch_one_resource(Realm, name=realmname)
     ret = realm.delete()
```

Comment:
Why not do this as a one-liner?

Reply:
fixed this in 708fb4c25cb559eb1cecfe3be209642ea2c3ef3a